src/routes/(auth)/connexion/+page.svelte

```diff
@@ -11,22 +12,26 @@
 <div class="container login">
 	<form method="POST" use:enhance>
 		<div class="form-widget">
-			<h1 class="header">Me connecter</h1>
+			<h1 class="header">J'ai déjà un compte</h1>
 			<Input
 				name="email"
 				type="email"
-				placeholder="Your email"
+				placeholder="Votre email"
 				value={form?.email ?? ''}
-				error={form?.email_error}>Email</Input
+				error={form?.email_error}>Email<span>&#42;</span></Input
 			>
 			<Input
 				value={form?.password ?? ''}
 				name="password"
 				type="password"
+				placeholder="Votre mot de passe"
 				required
-				error={form?.password_error}>Mot de passe</Input
+				error={form?.password_error}>Mot de passe<span>&#42;</span></Input
 			>
-			<Button>Me connecter</Button>
+			<Button>Connexion</Button>
+			<p>Première connexion ? <a href="inscription" class="connexionPath">Créer mon compte</a></p>
```

Comment:
Doest not look like the figma. Missing some spaces between elements and for the `retour` link, he could match the inscription design (flex row).

<img width="369" alt="Capture d’écran 2023-02-25 à 15 39 48" src="https://user-images.githubusercontent.com/46904617/221362951-d0773fc2-dff2-4d78-af37-fa7a1d4ad25e.png">